### PR TITLE
[now-cli] Fix links output during deployment

### PR DESCRIPTION
**Pull request by rdev**
This fixes deployment link not being reported on creation and double `https://` bug in build error message

### CODE REVIEW

File: packages/now-cli/src/util/deploy/process-deployment.ts [hunk 1]
```diff
@@ -81,6 +82,12 @@ export default async function processDeployment({
 
       if (event.type === 'created') {
         now._host = event.payload.url;
+
+        if (!quiet) {
+          log(`${event.payload.url} ${chalk.gray(`[v2]`)} ${deployStamp()}`);
```
Comment (styfle) [inline packages/now-cli/src/util/deploy/process-deployment.ts]:
```suggestion
          log(`https://${event.payload.url} ${chalk.gray(`[v2]`)} ${deployStamp()}`);
```

File: packages/now-cli/src/util/deploy/process-deployment.ts [hunk 2]
```diff
@@ -170,12 +177,16 @@ export default async function processDeployment({
         if (!quiet) {
           log(`Synced ${event.payload.size} ${uploadStamp()}`);
```
Comment (styfle) [inline packages/now-cli/src/util/deploy/process-deployment.ts]:
```suggestion
          log(`Synced ${event.payload.size} file(s) ${uploadStamp()}`);
```

File: packages/now-cli/src/util/deploy/process-deployment.ts [hunk 3]
```diff
@@ -37,6 +38,10 @@ export default async function processDeployment({
           `Total files ${event.payload.total.size}, ${event.payload.missing.length} changed`
         );
 
+        if (!quiet) {
+          log(`Synced ${event.payload.missing.length} ${uploadStamp()}`);
```
Comment (styfle) [inline packages/now-cli/src/util/deploy/process-deployment.ts]: Ignore what I wrote and use Nate's below

File: packages/now-cli/src/util/deploy/process-deployment.ts [hunk 4]
```diff
@@ -129,6 +134,9 @@ export default async function processDeployment({
         debug(
           `Total files ${event.payload.total.size}, ${event.payload.missing.length} changed`
         );
+        if (!quiet) {
+          log(`Synced ${event.payload.missing.length} ${uploadStamp()}`);
```
Comment (styfle) [inline packages/now-cli/src/util/deploy/process-deployment.ts]: Ignore what I wrote and use Nate's below

File: packages/now-cli/src/util/deploy/process-deployment.ts (hunk 3 above)
Comment (TooTallNate) [inline packages/now-cli/src/util/deploy/process-deployment.ts]:
```suggestion
          log(`Synced ${pluralize('file', event.payload.missing.length, true)} ${uploadStamp()}`);
```

(`pluralize()` might need to be imported)

File: packages/now-cli/src/util/deploy/process-deployment.ts (hunk 4 above)
Comment (TooTallNate) [reply packages/now-cli/src/util/deploy/process-deployment.ts]:
```suggestion
          log(`Synced ${pluralize('file', event.payload.missing.length, true)} ${uploadStamp()}`);
```

(`pluralize()` might need to be imported)

File: packages/now-cli/src/util/deploy/process-deployment.ts (hunk 3 above)
Comment (TooTallNate) [reply packages/now-cli/src/util/deploy/process-deployment.ts]:
```suggestion
          log(`Synced ${pluralize(event.payload.missing.length, 'file', true)} ${uploadStamp()}`);
```

File: packages/now-cli/src/util/deploy/process-deployment.ts (hunk 1 above)
Comment (styfle) [reply packages/now-cli/src/util/deploy/process-deployment.ts]: @rdev I'm pretty sure this needs to have https:// prefix to be backwards compatible with what it shows on stable. I'm not sure if the quiet part also needs it, but probably.

**Review by TooTallNate — DISMISSED** [review]
Can you post examples of what the incorrect output looks like that this is fixing? It's not entirely clear to me from just the diff.

Also, needs regression tests if possible.

File: packages/now-cli/src/util/deploy/process-deployment.ts (hunk 1 above)
Comment (rdev) [reply packages/now-cli/src/util/deploy/process-deployment.ts]: @styfle It's part of the payload now. This was causing the double `https://`

**Comment by rdev** [comment]
@TooTallNate 
The deployment URL not being shown on creation and double `https://` in error messages

image (5)

File: packages/now-cli/src/util/deploy/process-deployment.ts (hunk 1 above)
Comment (styfle) [reply packages/now-cli/src/util/deploy/process-deployment.ts]: I tried your branch again and I see the following:

```
> Deploying ~/Code/foo/node-try-catch under now-builders-ci-bot-85
> Using project node-try-catch
> node-try-catch-r4gd5xso2.now.sh [v2] [2s]
```

Notice how there is no https prefix. This is different behavior than `now@16.1.3`.

When I changed line 92, it printed the correct output.

**Review by styfle — CHANGES_REQUESTED** [review]
Please fix the URL from my comment https://github.com/zeit/now/pull/2966/files#r322385448

**Review by styfle — APPROVED** [review]

**Comment by TooTallNate** [comment]
The tests are still failing:

```
  TypeError [ERR_INVALID_URL] (NodeError) {
    input: '96i9pm7gevc-upuvdvajaa.now.shhttps://96i9pm7gevc-upuvdvajaa.now.sh',
    message: 'Invalid URL: 96i9pm7gevc-upuvdvajaa.now.shhttps://96i9pm7gevc-upuvdvajaa.now.sh',
  }
```

**Pull request merged**